jax_unirep/optimizers.py

```diff
@@ -0,0 +1,43 @@
+import jax.numpy as np
+from jax.experimental.optimizers import make_schedule, optimizer
+
+
+@optimizer
+def adamW(step_size, b1=0.9, b2=0.999, eps=1e-8, w=0.01):
+    """Construct optimizer triple for Adam.
+
+    Args:
```

Comment:
Leaving a note here why the docstring looks so different.

```suggestion
    This docstring is different from the rest because we want to submit this
    to the jax library, so DON'T CHANGE IT TO SPHINX-STYLE!

    Args:
```